crates/hera-exex/src/exex.rs

```diff
@@ -0,0 +1,60 @@
+//! Module for the Hera Execution Extension.
+
+use std::sync::Arc;
+
+use eyre::{bail, Result};
+use reth_exex::{ExExContext, ExExEvent};
+use reth_node_api::FullNodeComponents;
+use superchain_registry::RollupConfig;
+use tracing::{debug, info};
+
+use crate::cli::HeraArgsExt;
+
+/// The Hera Execution Extension.
+#[derive(Debug)]
+pub struct HeraExEx<Node: FullNodeComponents> {
```

Comment:
What do you think about this being the rollup `Driver` and creating a wrapper around the `ExExContext` which we can abstract here with a custom trait?